charts/refinery/templates/configmap-config.yaml

```diff
@@ -6,4 +6,36 @@ metadata:
   {{- include "refinery.labels" . | nindent 4 }}
 data:
   config.yaml: |
-{{- toYaml .Values.config | nindent 4 }}
+    EnvironmentCacheTTL: {{ .Values.config.EnvironmentCacheTTL }}
```

Comment:
Missing properties not mapped from values yaml:
- BatchTimeout
- AdditionalErrorFields
- HoneycombLogger (with sub properties)
- HoneycombMetrics (with sub properties)

Reply:
I'm new to this repo, but I'm a little concerned about listing out each `.Values.config` option in the config map.  If we add new config options in the future then we have to add them both to the values.yaml and in this file.  Instead, can we use some tpl helpers to handle the new `MaxAlloc` logic?  We should be able to check if the field is set in config using `get`.  [We do something similar in the opentelemetry-helm-charts.](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/fe8ee564b0c6c76f14e3af20daf318ae28855dce/charts/opentelemetry-collector/templates/_config.tpl#L26-L35)

Reply:
@TylerHelmuth, I had the same concern, but didn't know already of a good way to do this with template helpers. Thanks for the example -- will take a crack at that.

Reply:
@bixu no problem.  Inside the [opentelemetry-helm-charts](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/fe8ee564b0c6c76f14e3af20daf318ae28855dce/charts/opentelemetry-collector/templates/_config.tpl) there are some helper templates that do some calculating.  The `"opentelemetry-collector.convertMemToMib"` definition and `"opentelemetry-collector.getMemLimitMib"` will hopefully be helpful.

Reply:
Sorry @bixu I might have misunderstood your question.  You commit looks like it is doing a calculation if `.Values.config.InMemCollector "MaxAlloc"` is not set.  Is the calculation not working as expected?